questions.json

```diff
@@ -13,6 +14,22 @@
     "What is your least favorite friend's middle name?",
     "What is a question an oyster would ask a fish?",
     "What would your full name be if you married your first crush?",
-    "What caused you to realize that capitalism is a sham?"
+    "What caused you to realize that capitalism is a sham?",
+    "What is your favorite security question?",
+    "What are you wearing at time time of setting up the security question?",
+    "What is your password?",
+    "Which college class did finally break you?"
+    "What is the maiden name of your father's mistress?",
+    "At what age did your childhood pet run away?",
+    "What TV show do you keep rewatching to suppress the anxious feelings of the uncertainty of real life?",
+    "What was the name of your favorite unpaid internship?",
+    "What is the name of your favorite canceled TV show?",
+    "On which street did you lose your childlike sense of wonder?",
+    "In what year did you abandon your dreams?",
+    "When did you stop trying?",
+    "What is the phone number of your first crush that you still remember from 2007?",
+    "Which household appliance do you hit the most often?",
+    "Why is it always raining?",
+    "You should have said something. Back then! But what?"
```

Comment:
Maybe reword this to, "You should have said something back then. What should you have said?"